### ISSUE TITLE
Error adding entities for domain sensor with platform uptime_kuma

### ISSUE THREAD

**Issue by JOHLC** [meichthys/uptime_kuma#7]
Hello,

Thanks for your work on this component!

I am getting this error on Home Assistant, using the latest versions. I am not sure what this is affecting, sensors appear to populate/work. 
This issue has occured at least since 2022.6. 

Uptime Kuma component version: 1.1.1
Uptime Kuma version: 1.17.1
````
2022-07-09 11:40:11 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform uptime_kuma
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 397, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 548, in _async_add_entity
    original_icon=entity.icon,
  File "/config/custom_components/uptime_kuma/sensor.py", line 77, in icon
    return SENSORS_INFO[self.monitor.monitor_status]["icon"]
KeyError: 2.0
2022-07-09 11:40:11 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up uptime_kuma platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 258, in _async_setup_platform
    await asyncio.gather(*pending)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 397, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 548, in _async_add_entity
    original_icon=entity.icon,
  File "/config/custom_components/uptime_kuma/sensor.py", line 77, in icon
    return SENSORS_INFO[self.monitor.monitor_status]["icon"]
KeyError: 2.0
2022-07-09 11:40:15 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.141 seconds (success: True)
2022-07-09 11:40:25 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.148 seconds (success: True)
2022-07-09 11:40:35 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.136 seconds (success: True)
2022-07-09 11:40:45 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.157 seconds (success: True)

````

Home Assistant information

````
Home Assistant 2022.7.2
Supervisor 2022.07.0
Operating System 8.2
Frontend 20220707.0 - latest


Version	core-2022.7.2
Installation Type	Home Assistant OS
Development	false
Supervisor	true
Docker	true
Virtual Environment	false
Python Version	3.10.5
Operating System Family	Linux
Operating System Version	5.15.45
CPU Architecture	x86_64
Home Assistant Supervisor
Host Operating System	Home Assistant OS 8.2
Update Channel	stable
Supervisor Version	supervisor-2022.07.0
Agent Version	1.2.1
Docker Version	20.10.14
Disk Total	97.7 GB
Disk Used	23.3 GB
Healthy	true
Supported	true
Supervisor API	ok
Version API	ok
````

**Comment by meichthys** [comment]
@jayakornk, could you take a look at this? I haven't had a chance to review the code changes yet.

**Comment by jayakornk** [comment]
> Hello,
> 
> Thanks for your work on this component!
> 
> I am getting this error on Home Assistant, using the latest versions. I am not sure what this is affecting, sensors appear to populate/work. This issue has occured at least since 2022.6.
> 
> Uptime Kuma component version: 1.1.1 Uptime Kuma version: 1.17.1
> 
> ```
> 2022-07-09 11:40:11 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform uptime_kuma
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 397, in async_add_entities
>     await asyncio.gather(*tasks)
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 548, in _async_add_entity
>     original_icon=entity.icon,
>   File "/config/custom_components/uptime_kuma/sensor.py", line 77, in icon
>     return SENSORS_INFO[self.monitor.monitor_status]["icon"]
> KeyError: 2.0
> 2022-07-09 11:40:11 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up uptime_kuma platform for sensor
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 258, in _async_setup_platform
>     await asyncio.gather(*pending)
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 397, in async_add_entities
>     await asyncio.gather(*tasks)
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 548, in _async_add_entity
>     original_icon=entity.icon,
>   File "/config/custom_components/uptime_kuma/sensor.py", line 77, in icon
>     return SENSORS_INFO[self.monitor.monitor_status]["icon"]
> KeyError: 2.0
> 2022-07-09 11:40:15 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.141 seconds (success: True)
> 2022-07-09 11:40:25 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.148 seconds (success: True)
> 2022-07-09 11:40:35 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.136 seconds (success: True)
> 2022-07-09 11:40:45 DEBUG (MainThread) [custom_components.uptime_kuma] Finished fetching uptime_kuma data in 0.157 seconds (success: True)
> ```
> 
> Home Assistant information
> 
> ```
> Home Assistant 2022.7.2
> Supervisor 2022.07.0
> Operating System 8.2
> Frontend 20220707.0 - latest
> 
> 
> Version	core-2022.7.2
> Installation Type	Home Assistant OS
> Development	false
> Supervisor	true
> Docker	true
> Virtual Environment	false
> Python Version	3.10.5
> Operating System Family	Linux
> Operating System Version	5.15.45
> CPU Architecture	x86_64
> Home Assistant Supervisor
> Host Operating System	Home Assistant OS 8.2
> Update Channel	stable
> Supervisor Version	supervisor-2022.07.0
> Agent Version	1.2.1
> Docker Version	20.10.14
> Disk Total	97.7 GB
> Disk Used	23.3 GB
> Healthy	true
> Supported	true
> Supervisor API	ok
> Version API	ok
> ```

Which monitor type are you using? I might have to add more `SENSORS_INFO` config to support them.

**Comment by JOHLC** [comment]
Hello,

I am using multiple monitor types, including: DNS, ping, HTTP(S), TCP port, and MQTT.

**Comment by jayakornk** [comment]
I will try out DNS and TCP port. Maybe I can find something interesting. 😬

**Comment by jayakornk** [comment]
I'm using **Home Assistant 2022.7.3 (Container)** and **Uptime Kuma 1.17.1** but everything seem to work ok without any error.

My monitor includes HTTP, Keyword, TCP port, DNS, MQTT, and Ping.

**Comment by meichthys** [comment]
> **Uptime Kuma 1.17.1** ?

I'm only seeing a v1.1.1

**Comment by jayakornk** [comment]
I mean version of Uptime Kuma not this integration.
<img width="264" alt="image" src="https://user-images.githubusercontent.com/42465929/178310957-65ff461b-26bc-4d1d-b76c-eec65d8f0a66.png">

**Comment by jayakornk** [comment]
Wait for https://github.com/meichthys/uptime_kuma/pull/9 to merge and create new release (Integration v1.2.1)

**Comment by meichthys** [comment]
Merged and released in v1.2.0 👍
